api/index.ts

```diff
@@ -17,6 +17,7 @@ export interface MarketStatsResponse {
 export default async (req: NowRequest, res: NowResponse) => {
   res.setHeader('Access-Control-Allow-Origin', '*')
   res.setHeader('Access-Control-Allow-Methods', 'GET')
+  res.setHeader("Cache-Control", "max-age=0, s-maxage=600")
```

Comment:
Single quotes to make Travis happy:

```suggestion
  res.setHeader('Cache-Control', 'max-age=0, s-maxage=600')
```